lib/compress.js

```diff
@@ -3028,63 +3028,49 @@ merge(Compressor.prototype, {
         return this;
     });
 
-    var commutativeOperators = makePredicate("== === != !== * & | ^");
+    var reversibleOperators = makePredicate("+ * & | ^ < > <= >= == != === !==");
```

Comment:
I think I'll just rip this all out and just revert 88fb83aa for now - that section of code won't work almost all the time.

#415 can wait for another time.